cookbook/securing-your-application.rst

```diff
@@ -180,6 +180,10 @@ needs some updating. You can add the permission tab as shown below:
         }
     }
 
+.. note::
+
+    The usage of the ``setRequestParameters`` method in the FormViewBuilder is deprecated. Please use ``addRequestParameters`` instead.
```

Comment:
can we update the docs just using the new parameter?

Reply:
Has been altered.